For klaw minor release 2.10.1 (#352)
Aiven-Open/klaw-docs: docs/releases/release2101.md

---
sidebar_position: 99.84
---

# Release 2.10.1

> Date: 6th of June, 2025

Reviewer: ```suggestion
Date: June 6, 2025
```

May be we should update all the dates in this format. Don't know how we missed this previously.

## Overview

> Overview
> Klaw version 2.10.1 is a patch release which improves couple of user flows while a user is onboarding with Single sign on.

Reviewer: ```suggestion
Klaw version 2.10.1 is a patch release that improves a few onboarding flows for single sign-on (SSO) users.
```
